CustomEvents.cpp

```diff
@@ -1059,6 +1059,10 @@ bool CustomEventsParser::ParseCustomEvent(rapidxml::xml_node<char> *node, Custom
                 else if (strcmp(stringValue, "player") == 0) {
                     customEvent->recallBoardersShip = -1;
                 }
+            }
+			if (child->first_attribute("force"))
```

Comment:
This line uses the wrong kind of indentation, should be four spaces instead of the tab character.